packages/component/src/index.ts

```diff
@@ -103,23 +105,27 @@ const Components = {
   UploadButton,
 
   // Components for localization
-  LocalizedString
+  LocalizedString,
+
+  // Components for theming
+  MonochromeImageMasker
```

Comment:
Should we include some docs?

Reply:
Oh, maybe I should tag it as `internal`.

Reply:
Can't put it in `internal` yet as Jest run OOM while loading `botframework-webchat-api`.